### ISSUE TITLE
[General UI] Text color of focused tab in tabs block has too little contrast

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#410]
### Steps to reproduce the problem

1. Go to https://learn.qiskit.org/course/introduction/project
2. Scroll down to the "Step 3: Decide on a project" section and to the "Project ideas" tabs block
3. Focus (for example by clicking the "tab" key) on the tabs
4. Attempt to read the label

### What is the current behavior?

When focused, the text color is white, which is almost impossible to read.

### What is the expected behavior?

The text should have enough contrast when focused.

Even keeping the same color as when not hovered might be a good solution since, when tabbing, a border already indicates the currently focussed element.

### Screenshots (optional but very helpful)

![image](https://user-images.githubusercontent.com/22047320/140353328-bf5d1e20-4a05-4b64-b252-695b5728aeee.png)

### Environment (optional but very helpful)
 - Device: Laptop
 - OS: MacOS
 - Browser: Chrome

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
